ext/droption/droption.h

```diff
@@ -539,6 +540,20 @@ droption_t<int>::convert_from_string(const std::string s)
 }
 template <>
 inline bool
+droption_t<long>::convert_from_string(const std::string s)
+{
+    value = atol(s.c_str());
```

Comment:
Should we use strtol so we get error detection?  Ditto below (and in the original code looks like).